Makefile

```diff
@@ -19,3 +19,9 @@ local:
 
 start_all:
```

Comment:
Think about that across your PR. I know the make targets are defined one way, unclear whether its the best way